src/functionalTest/resources/features/F-017/F-017.feature

```diff
@@ -0,0 +1,89 @@
+    Then a negative response is received,
+    And the response has all other details as expected
+
+  @S-253
+  Scenario: Invoking POST role-assignments api with request to add a new role assignment but the process does not match 'professional-organisational-role-mapping'
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    When a request is prepared with appropriate values,
+    And the request [has roleType Organisation, roleCategory Professional, roleName Role1 and process value is not set to 'professional-organisational-role-mapping'],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a negative response is received,
+    And the response has all other details as expected
+
+  @S-254
+  Scenario: Invoking POST role-assignments api with request to add a new role assignment with not valid classification
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    When a request is prepared with appropriate values,
+    And the request [has roleType Organisation, roleCategory Professional, roleName Role1 and Classification is not set to Restricted],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a negative response is received,
+    And the response has all other details as expected
+
+  @S-255
+  Scenario: Invoking POST role-assignments api with request to add a new role assignment with not valid grantType
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    When a request is prepared with appropriate values,
+    And the request [has roleType Organisation, roleCategory Professional, roleName Role1 and grantType not set to STANDARD],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a negative response is received,
+    And the response has all other details as expected
+
+  @S-256
+  Scenario: Invoking GET role assignment with valid actorId and returns 200 OK
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Befta1 - who invokes the API],
+    And a successful call [to create org role assignments for actors & requester] as in [S-256_Org_Role_Creation],
+    When a request is prepared with appropriate values,
+    And the request [has a actor id passed that does exists],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a positive response is received,
+    And the response has all other details as expected
+    And a successful call [to delete role assignments just created above] as in [S-250_DeleteDataForRoleAssignmentsForOrgRoles].
```

Comment:
Delete role assignments won't work due to reference value is not matched:

[S-256_Org_Role_Creation]:
 "process" : "professional-organisational-role-mapping",
  "reference" : "cb19a220-1eb3-4da9-88a0-95e21fc1ff3d",

[S-250_DeleteDataForRoleAssignmentsForOrgRoles]
  "pathVariables" : {
    "process" : "professional-organisational-role-mapping",
    "reference" : "37c1d803-14e9-4850-ad8f-7c387f8490cf"
  }

Can you please update  "reference" : "cb19a220-1eb3-4da9-88a0-95e21fc1ff3d" for delete?
You can create a separate json file for S-256 Deletion instead of using S-250 deletion data.